build.gradle

```diff
@@ -1080,6 +1080,17 @@ project('spring-integration-bom') {
     }
 }
 
+
+project('spring-integration-hazelcast') {
+    description = 'Spring Integration Hazelcast Support'
+    dependencies {
+	implementation "com.hazelcast:hazelcast:$hazelcastVersion"
```

Comment:
Why not `api`?

Reply:
Hmm I thought Spring integration _only_ used Hazelcast _internally_ and hence not exposing it as an Api, but I might be wrong here?
I did a quick analysis of the Spring integration Api and it seems that Hazelcast´s own Api is not leaked through. But maybe we want to expose its Api as well?

Reply:
Yeah... I know what you mean, but our goal is let end-user just add `spring-integration-hazelcast` dependency and `hazelcast` by itself is going to be pulled transitively.
The `api` is an equivalent of `compile` in Maven, where an `implementation` is mapped to `runtime`.
See more info here: https://stackoverflow.com/questions/44413952/gradle-implementation-vs-api-configuration

Reply:
Ok fair enough I will change that